### PR TITLE
Tweaks AI door open and close buttons

### PR DESCRIPTION
**Pull request by JJRcop**
:cl: JJRcop
tweak: The silicon airlock menu looks a little more like it used to.
/:cl:

sometimes_people_forget

### CODE REVIEW

**Comment by AnturK** [comment]
Already in #32737

**Comment by JJRcop** [comment]
Oops... Well, this one is, better? Guess it's debatable.

**Comment by JJRcop** [comment]
@AnturK could you as'bsorb this PR into #32737? If not I'll PR it to your branch in the morning.

**Comment by AnturK** [comment]
Sorry, just got merged. Feel free to overwrite my button though.

**Comment by JJRcop** [comment]
This isn't a fix anymore.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1385,6 +1385,8 @@
 	data["lights"] = lights // bolt lights
 	data["safe"] = safe // safeties
 	data["speed"] = normalspeed // safe speed
+	data["welded"] = welded // welded
```
Comment (JJRcop) [inline code/game/machinery/doors/airlock.dm]: Oops https://github.com/tgstation/tgstation/blob/ef21d9847835018466a7a938991361b8e91e1365/code/game/machinery/doors/airlock.dm#L1379

**Pull request merged**